src/MonthHW3.java

```diff
@@ -0,0 +1,50 @@
+public class MonthHW3 {
+    public static void getDaysInMonth(int monthNumber) {
+        switch (monthNumber) {
+            case 1:
```

Comment:
static int monthDayQuantity(int monthNumber) {
        switch (monthNumber) {
            case 1:
            case 3:
            case 5:
            case 7:
            case 8:
            case 10:
            case 12:
                System.out.println("31");
                break;
            case 2:
                System.out.println("28");
                break;
            case 4:
            case 6:
            case 9:
            case 11:
                System.out.println("30");
                break;
            default:
                System.out.println("No month with such number");
        }
        return monthNumber;
    }

Reply:
It's a good idea to optimize the code

Reply:
Use English